content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/configuring-your-minio-storage-bucket-for-github-actions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Configuring your MinIO storage bucket for GitHub Actions
+intro: 'Configure your custom MinIO storage bucket for use with {% data variables.product.prodname_actions %}.'
+versions:
+  ghes: '*'
+type: quick_start
+topics:
+  - Actions
+  - Enterprise
+  - Infrastructure
+  - Storage
+shortTitle: Quickstart for MinIO
+---
+
+Before you can enable and configure {% data variables.product.prodname_actions %} on {% data variables.product.product_location_enterprise %}, you need to prepare your third-party storage solution.
+
+MinIO offers object storage with support for the S3 API and {% data variables.product.prodname_actions %} on your enterprise.
+
+This quickstart shows you how to set up MinIO using Docker for use with {% data variables.product.prodname_actions %} but you have other options for managing MinIO besides Docker. For more information about MinIO, see the official [MinIO docs](https://docs.min.io/).
+
+## 1. Choose a MinIO mode for your needs
+
+| MinIO mode | Optimized for | Storage infrastructure required |
+|----|----|----|
+| Standalone MinIO (on a single host) | Fast setup |  N/A |
+| Clustered MinIO (also called Distributed MinIO)|  Data security | Storage servers running in a cluster |
+
+For more information about your options, see the official [MinIO docs](https://docs.min.io/).
+
+## 2. Install, run, and sign in to MinIO
+
+1. Set up your preferred environment variables for MinIO.
+
+    These examples use `MINIO_DIR`:
+    ```shell
+    $ export MINIO_DIR=$(pwd)/minio
+    $ mkdir -p $MINIO_DIR
+    ```
+
+2. Install MinIO.
+
+    ```shell
+    $ docker pull minio/minio
+    ```
+    For more information, see the official "[MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide)."
+
+3. Sign in to MinIO using your MinIO access key and secret.
+
+    {% linux %}
+    ```shell
+    $ export MINIO_ACCESS_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
probably we should write about the new environment variable MINIO_ROOT_USER and MINIO_ROOT_PASSWORD 

these ones are legacy.

Reply:
We are removing this file altogether.